.circleci/config.yml

```diff
@@ -150,6 +187,40 @@ jobs:
             export CXX="c++"
             ../../build.sh "-D__BLST_PORTABLE__"
             ./run.me
+
+            mkdir portable
+            mv supranational/blst/Mac/* portable/
+
+      - run:
+          name: Generate mac os shared lib (Optimised)
+          command: |
+            echo "Building for arm64"
+            cd blst/bindings/java
+            rm -rf libblst.a supranational.blst.jar
+            rm -rf supranational
```

Comment:
we'd need an x86 and M1 build for mac depending on arch?

Reply:
Yep that's what it's building.  So there wind up being 4 libraries build for Mac.

```
     0 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/
     0 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/x86_64/
251016 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/x86_64/libblst.dylib
     0 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/optimised/
     0 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/optimised/aarch64/
252842 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/optimised/aarch64/libblst.dylib
     0 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/optimised/x86_64/
251016 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/optimised/x86_64/libblst.dylib
     0 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/aarch64/
252842 Thu Dec 02 14:07:00 AEST 2021 supranational/blst/Mac/aarch64/libblst.dylib
```

I'm not sure if there is actually a difference for ARM cpus but it makes life a lot simpler to just build both versions for all architectures.